### PR TITLE
Adrv9002 interface gain fix

### PR DESCRIPTION
**Pull request by SrikanthPagadarai**

### CODE REVIEW

File: +adi/+ADRV9002/Rx.m [hunk 1]
```diff
@@ -221,15 +221,13 @@
             'automatic','spi'});
         DigitalGainControlModeChannel1Set = matlab.system.StringSet({ ...
             'automatic','spi'});
-        
-        InterfaceGainChannel0Set = matlab.system.StringSet({...
-            '18dB', '12dB', '6dB', '0dB', '-6dB', '-12dB', '-18dB',...
-            '-24dB', '-30dB','-36dB'});
-        InterfaceGainChannel1Set = matlab.system.StringSet({...
-            '18dB', '12dB', '6dB', '0dB', '-6dB', '-12dB', '-18dB',...
-            '-24dB', '-30dB','-36dB'});
```
Comment (tfcollins) [inline +adi/+ADRV9002/Rx.m:230]: Why are these removed?

**Review by tfcollins — COMMENTED** [review]
Need to update the interface gain controls in setupInit as well.

Can you enhance the doc tell the user the interface gain is profile dependent as well

File: +adi/+ADRV9002/Rx.m (hunk 1 above)
Comment (SrikanthPagadarai) [reply +adi/+ADRV9002/Rx.m:230]: Updated.

**Comment by SrikanthPagadarai** [comment]
> Need to update the interface gain controls in setupInit as well.
> 
> Can you enhance the doc tell the user the interface gain is profile dependent as well

Updated setupInit and doc-strings.

**Review by tfcollins — APPROVED** [review]
